plugins/modules/database/postgresql/postgresql_user.py

```diff
@@ -38,8 +38,8 @@
       plain text format.
     - When passing an MD5-hashed password, you must generate it with the format
       C('str["md5"] + md5[ password + username ]'), resulting in a total of
-      35 characters. An easy way to do this is C(echo "md5$(echo -n
-      'verysecretpasswordJOE' | md5sum | awk '{print $1}')").
+      35 characters. An easy way to do this is
+      ``echo "md5$(echo -n 'verysecretpasswordJOE' | md5sum | awk '{print $1}')"``.
```

Comment:
This won't work. ansible-doc does not support RST formatting, and for the HTML docs this ends up in a HTML blob, not in a real RST environment.

I think it's best to use single backticks instead of `$(...)`.